### PR TITLE
fix: socket updates on enter, skipping current user

### PR DESCRIPTION
**Pull request by mingard**

### CODE REVIEW

File: frontend/components/DocumentEditToolbar/DocumentEditToolbar.jsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ export default class DocumentListToolbar extends Component {
 
     return (
       <Toolbar>
-        {peers && peers.length &&
+        {Boolean(peers && peers.length) &&
```
Comment (eduardoboucas) [inline frontend/components/DocumentEditToolbar/DocumentEditToolbar.jsx]: Why the `Boolean()`?

Comment (mingard) [inline frontend/components/DocumentEditToolbar/DocumentEditToolbar.jsx]: When `peers` is length 0 the condition `peers.length` is still a valid condition, meaning the output in the dom is just `0`.

Comment (eduardoboucas) [inline frontend/components/DocumentEditToolbar/DocumentEditToolbar.jsx]: Ah okay. I think `peers && (peers.length > 0)` would be more descriptive, but I guess that's just personal taste.

Comment (mingard) [inline frontend/components/DocumentEditToolbar/DocumentEditToolbar.jsx]: I'll change to that

**Pull request merged**